components/ChoreFrequency.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { Badge, Card, Text } from 'react-native-paper';
+
+const ChoreFrequency = () => {
```

Comment:
For consistency write component as a function
```suggestion
export default function ChoreFrequency() {
```

Reply:
fixed